### DISCUSSION TITLE
Option for set loop point in sound2 (and other parts if possible)

### DISCUSSION THREAD

**Discussion by lilggamegenius** [CapsAdmin/pac3#666] [Ideas]
If its possible to add, and option to set what sample to loop at when the end of the sound is reached, if the sound is set to loop. This should default to 0, looping from beginning and also keeping backwards compatibility.

**Comment by thegrb93** [comment]
Afaik it's not possible except for a hacky think hook implementation or modifying the sound file itself. I don't know how to the javascript portion of the sound playback works though so there might be some possibility there.

**Comment by CapsAdmin** [comment]
It's easy to implement I think. I'll try to look into it when I have time.

**Comment by notcat** [comment]
#666 monkaS

**Comment by lilggamegenius** [comment]
what
